src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -40,6 +40,12 @@
     </Compile>
 
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' "> 
+ 
+    <Compile Include="SslStreamStreamToStreamTest.cs" />   
+    <Compile Include="TestConfiguration.cs" />    
+       
+  </ItemGroup>
```

Comment:
I've lost track of what the CI system is capable of on Linux/OSX currently, but up until very recently all of our test binaries were the same across platform, and then only recently did some such conditioning sneak in.  You may need to confirm with @mmitche that this is going to test what you want it to test.

Separately, why are these additional test files specific to Unix?  And why are the previous ones specific to Windows?  Is that just a moment-in-time thing, that those other tests don't pass yet so rather than marking them as ActiveIssue we're keeping all of those tests out of the build on Unix?